modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleLocalServiceImpl.java

```diff
@@ -170,6 +226,22 @@ public void deleteObjectValidationRules(Long objectDefinitionId)
 		}
 	}
 
+	@Override
+	public ObjectValidationRule fetchObjectValidationRule(
```

Comment:
Do we need this if we start using ObjectValidationResource in the Object Definition Resource?